agenta-backend/agenta_backend/tasks/evaluations.py

```diff
@@ -94,8 +94,18 @@ def evaluate(
     loop = asyncio.get_event_loop()
 
     try:
-        # 1. Fetch data from the database
         loop.run_until_complete(DBEngine().init_db())
+
+        # 0. Update evaluation status to STARTED
+        loop.run_until_complete(
+            update_evaluation(
+                evaluation_id,
+                {"status": Result(type="status", value="EVALUATION_STARTED")},
+            )
+        )
+        self.update_state(state=states.STARTED)
```

Comment:
Is this redundant? I would expect celery to do that by itself, no?

Reply:
changing the config seems to be the right way here https://celery.school/custom-celery-task-states
![Uploading CleanShot 2024-07-05 at 10.39.45@2x.png…]()

Reply:
Removing line 106.
Adding config in config file.